programs/mango-v4/Cargo.toml

```diff
@@ -44,7 +44,7 @@ solana-sdk = { workspace = true, default-features = false, optional = true }
 solana-security-txt = "1.1.0"
 static_assertions = "1.1"
 switchboard-program = ">=0.2.0"
-switchboard-v2 = "0.1.17"
+switchboard-v2 = { git = "https://github.com/switchboard-xyz/solana-sdk.git", tag = "sbv2-crate-0.4.0" }
```

Comment:
switchboard-v2 seems to have the 0.4.0 crate published - why do we need the git dependency?

Reply:
on [crates.io](https://crates.io/crates/switchboard-v2/0.4.0) it says:
> Deprecated in favor of the switchboard-solana crate.
this seems to offer a published version for anchor 28 - will try with that one again

Reply:
using this now:
`switchboard-v2 = { package = "switchboard-solana", version = "0.28.18" }`